installer/PowerToysSetup/PowerToys.wxs

```diff
@@ -113,10 +113,10 @@
                 RepairCommand="/repair /quiet /norestart"
                 Permanent="yes">
                 <RemotePayload
```

Comment:
Let's please add a `<ExitCode Value="1638" Behavior="success"/>` tag here, so this error gets ignored in the future. Similar to what we have earlier in this file closer to the .net payload. (0x666 = 1638)

Reply:
Done!